pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -65,7 +65,9 @@ type GCECloud struct {
 	service           *compute.Service
 	containerService  *container.Service
 	projectID         string
-	zone              string
+	region            string
+	instanceZone      string
+	managedZones      []string
```

Comment:
Can the "managed zones" be dynamically inferred from the location of the nodes that have registered with the cluster?